src/components/DatePicker/index.android.js

```diff
@@ -4,19 +4,31 @@ import moment from 'moment';
 import TextInput from '../TextInput';
 import CONST from '../../CONST';
 import {propTypes, defaultProps} from './datepickerPropTypes';
+import DateUtils from '../../libs/DateUtils';
 
 class DatePicker extends React.Component {
     constructor(props) {
         super(props);
-
+        const value = DateUtils.getDateAsText(props.value) || DateUtils.getDateAsText(props.defaultValue) || CONST.DATE.MOMENT_FORMAT_STRING;
```

Comment:
The problem with empty string is, the Labels are not active with an empty string and explicitly calling `activateLabel` can be a tad bit messy.Moreover defaulting to a standard 'DD/MM/YYYY' format will help us to be more consistent among devices because we already show format as a placeholder on the Web platform.

Reply:
Yea, I was thinking about that and how maybe we should remove that placeholder altogether and put the date format in the label instead. That's something that we agreed upon on the design doc (see doc quote below). That way the user can always check the date format as they type in the date. We'd need to make the label dynamic though, so that it can take different date formats depending on localization. 

>Placeholders should not be used as it’s customary for labels to appear inside form fields and animate them above the field when focused.

Reply:
Do we need to do that though? Because in Android/iOS devices, the `datepicker` is not editable and is based on a selection dialogue. It seems to be editable only on the Web, which shows the placeholder because of the select tag used in it.

Reply:
Hmm I think we should:

1. Leave native as is.
2. Remove the placeholder on Web/desktop and add that to the label.

Reply:
The Datepicker we have been using is the native Datepicker itself and it's not possible to decouple the placeholder from it as of now. To remove it we will have to add in a new Datepicker module for selection of date but I think it will be more apt to create a new Issue for that.

Reply:
Let's keep it as is.